tests/csm/rest/test_iam_users.py

```diff
@@ -424,14 +424,233 @@ def test_35930(self):
         self.log.info("[END]Creating Max IAM user with random selection of optional parameters")
         self.log.info("##### Test completed -  %s #####", test_case_name)
 
+    # pylint: disable=too-many-statements
     @pytest.mark.lc
     @pytest.mark.csmrest
-    @pytest.mark.cluster_user_ops
+    @pytest.mark.s3_ops
+    @pytest.mark.parallel
```

Comment:
I can see parallel marker added here, is this tested with parallel execution?

Reply:
removed